Program.cs

```diff
@@ -4,17 +4,11 @@
 string userinput = Console.ReadLine ();
 string input = userinput.ToLower ();
 List<char> order = new List<char> ();
-for (int i = 0; i < input.Length; i++) {
-   if (input[i] != ' ') {
-      order.Add (input[i]);
-   }
-}
+for (int i = 0; i < input.Length; i++) if (input[i] != ' ') order.Add (input[i]);

```

Comment:
You can use trim and tochararray to get the character in an array no need of this loop and order variable.